### PR TITLE
[board] basic support of matek h743 slim

### PR DESCRIPTION
**Pull request by gautierhattenberger**
only flashing with DFU for now
default orientation for MPU6000 imu
fix small bugs

still need drivers for ICM42605 imu and DPS310 baro
dshot config is missing

### CODE REVIEW

**Comment by OpenUAS** [comment]
Could flash ((no Bootloader but ST DFU direct) Matek h743 slim, have data out nicely via USB but no MPU6000 data (SPI1 AFAIK) also not raw

**Comment by gautierhattenberger** [comment]
The IMU on spi1 (MPU6000) is just working fine with my test board. You tried with the demo airframe file ?

**Comment by OpenUAS** [comment]
Yes, but will retry.

**Comment by OpenUAS** [comment]
I would say merge it with master already, easier for others to fix along the way. Anyhow:

V1, IMU MPU6000, ICM20602
V1.5, IMU MPU6000, ICM42605
V3,  IMU ICM42688P, ICM42605

File: conf/boards/matek_h743_slim.makefile [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Hey Emacs, this is a -*- makefile -*-
+#
+# cube_orange.makefile
```
Comment (fvantienen) [inline conf/boards/matek_h743_slim.makefile:3]: Rename to the correct filename

Comment (fvantienen) [inline conf/boards/matek_h743_slim.makefile:3]: Rename to the correct filename

File: sw/airborne/boards/mateksys/FC-H743-SLIM/mcuconf.h [hunk 2]
```diff
@@ -0,0 +1,593 @@
...
+    limitations under the License.
+*/
+
+#ifndef MCUCONF_H
+#define MCUCONF_H
+
+/*
+ * Enforce old versions of the chip
+ */
+#define STM32_ENFORCE_H7_REV_XY
```
Comment (fvantienen) [inline sw/airborne/boards/mateksys/FC-H743-SLIM/mcuconf.h:23]: Are you sure they use old versions?

**Review by fvantienen — APPROVED** [review]
Just some minor things, but looks good! Nice that you also use the common board

**Comment by fvantienen** [comment]
For the newer inversense drivers see https://github.com/jesusBV20/paparazzi_jesusBV20/commit/8871e70ef3df0353f903182b2f65602087a99bb6

File: conf/boards/matek_h743_slim.makefile (hunk 1 above)
Comment (gautierhattenberger) [reply conf/boards/matek_h743_slim.makefile]: fixed

File: sw/airborne/boards/mateksys/FC-H743-SLIM/mcuconf.h (hunk 2 above)
Comment (gautierhattenberger) [reply sw/airborne/boards/mateksys/FC-H743-SLIM/mcuconf.h:23]: I recall checking for that and ardupilot is also using this flag

**Pull request merged**